lexicons/bsky.app/invite.json

```diff
@@ -0,0 +1,22 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.invite",
+  "type": "record",
+  "key": "tid",
+  "record": {
+    "type": "object",
+    "required": ["group", "subject", "createdAt"],
+    "properties": {
+      "group": {"type": "string"},
```

Comment:
Since the invite is created _by_ the group, we don't need to include the declaration CID as a check. 

However, you see that we do include the declaration CID on `inviteAccept`.